radixdlt-core/radixdlt/src/main/java/com/radixdlt/sync/CommittedReader.java

```diff
@@ -30,4 +30,23 @@
 	VerifiedTxnsAndProof getNextCommittedTxns(DtoLedgerProof start);
 	Optional<LedgerProof> getEpochProof(long epoch);
 	Optional<LedgerProof> getLastProof();
+
+	static CommittedReader mocked() {
```

Comment:
not a big fan of leaking test classes to `src/main`

Reply:
yeah this is quite a hack but it's required for GenerateUniverses, not just test classes